conf/test.config

```diff
@@ -23,12 +23,16 @@ params {
     input = 'https://raw.githubusercontent.com/nf-core/test-datasets/chipseq/samplesheet/v2.0/samplesheet_test.csv'
 
     // Genome references
-    fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/reference/genome.fa'
-    gtf   = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/reference/genes.gtf'
+    genome = 'hg19'
+    // fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/reference/genome.fa'
+    // gtf   = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/reference/genes.gtf'
```

Comment:
These are commented out?

Reply:
Indeed, that is a leak from some test I run on my local machine 😓

Reply:
Fixed in [277bd11](https://github.com/nf-core/chipseq/pull/279/commits/277bd119f4747960d0059df30f7a3cea8fccb625)